packages/conda-store/src/condaStoreEnvironmentManager.ts

```diff
@@ -119,12 +120,26 @@ export class CondaStoreEnvironmentManager implements IEnvironmentManager {
     return;
   }
 
+  /**
+   * Create a new conda-store environment by importing a YAML-formatted environment file.
+   *
+   * Valid examples of environment files can be found in the conda-store and conda documentation; at
+   * a minimum, a valid file must contain a name and a list of conda package dependencies.
+   *
+   * @async
+   * @param {string} name - Name of the namespace/environment name to create.
+   * @param {string} fileContent - Raw contents of the file to import.
+   * @param {string} fileName - Name of the file to use for import.
+   * @returns {Promise<void>}
```

Comment:
Usually nothing is specified when a method returns void.

```suggestion
```

Reply:
Thanks, I've removed this instance and another in the same file.